src/geovista/pantry/data.py

```diff
@@ -24,8 +24,8 @@
 from numpy import ma
 import pooch
 
-from .cache import CACHE
-from .common import LRU_CACHE_SIZE, _MixinStrEnum
+from geovista.cache import CACHE
```

Comment:
Note that one of the pre-commit hooks (can't remember which one) objects to `from ..cache`, necessitating this change now that the module has moved into a subdirectory